.flake8

```diff
@@ -4,4 +4,4 @@ ignore = E203, E266, W503, ANN002, ANN003, ANN101, ANN102, ANN401, N807, N818
 max-line-length = 79
 max-complexity = 18
 select = B,C,E,F,W,T4,B9,ANN,Q0,N8,VNE
-exclude = venv, tests
+exclude = ve, tests
```

Comment:
Name your virtual environment as `venv` in the future so that there would be no need to adjust `.flake8` file